### ISSUE TITLE
Problem with fog and sky

### ISSUE THREAD

**Issue by Yberion** [mvdevs/jk2mv#176]
Hello,

Someone on JKCommunity's discord pointed out there might be a problem witht he following line:

- https://github.com/mvdevs/jk2mv/blob/master/src/renderer/tr_shader.cpp#L3354

It appear that the fix should be something like that:

```cpp
if ( stage == 0 && !shader.sky && !shader.isSky) {
```

I'm not sure about `!shader.isSky` as in the discord's conv they are talking about `!issky` and I don't know from where that comes from as it's not on jk2mv code.

[The explaination is the following](https://ptb.discord.com/channels/200934142278369281/401384051115687937/1279461000004894842):

```
it causes sky to be drawn in front of transparent elemtns 
which ruins stuff like speed ghosts
or applied ent alpha
...
basically the place where SS_FOG is applied to stuff
but it shouldnt be applied to sky
```
